sk0-back/src/users/services/auth.service.ts

```diff
@@ -33,7 +38,13 @@ export class AuthService {
     };
 
     let createdUser = await this.usersService.create(userToCreate);
+
+    this.notifyWith.sendWith(
+      this.mailer,
+      validationEmail(createdUser.email, createdUser.confirmation)
+    );
     createdUser = instanceToInstance(createdUser);
+
```

Comment:
I think inststead of reassigning the value we should rather be immutable and create another object and return with it. But I can see why you wanted to do this way. Although TyeScript should not allow the entity object dhape to return with.